src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/TargetMenuItemViewModel.cs

```diff
@@ -33,7 +33,7 @@ public static TargetMenuItemViewModel Create(InheritanceTargetItem target)
         {
             var targetName = target.DefinitionItem.DisplayParts.JoinText();
             var displayContent = string.Format(ServicesVSResources._0_in_1, targetName, target.DisplayNameForContainingType);
-            var imageMoniker = target.Glyph.GetImageMoniker();
+            var imageMoniker = InheritanceMarginHelpers.GetMoniker(target.RelationToMember);
```

Comment:
💭 I'm torn on which one I like more...

Reply:
So how about have both of them, the TypeImage is set at the beginning and the up/down icon is appended

Reply:
Mark this as resolved since the UI is modified in another way